src/components/BlogCard/BlogCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./BlogCard.scss";
+import defaultUserImage from "./../../assets/images/icons/default.png";
+
+const BlogCard = ({ blog }) => {
+  const { title, tags, image, createdAt } = blog;
+  return (
+    <div className="blog-card">
+      <img src={image} className="blog-card__banner" />
+      <div className="blog-card__category-badge">{tags}</div>
+      <h3 className="blog-card__blog-title">{title}</h3>
+      <div className="blog-card__author-info">
+        <img
+          src={defaultUserImage}
```

Comment:
will it be replaced with a real image later?

Reply:
yes vaiya